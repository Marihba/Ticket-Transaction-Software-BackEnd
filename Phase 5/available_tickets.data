My Awesome Event          event host      029 032.95
A new event!              admin           055 001.23
A new event!              admin           055 001.23
END                                                 