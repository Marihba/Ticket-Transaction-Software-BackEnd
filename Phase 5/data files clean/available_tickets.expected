A new event!              admin           055 001.23
END                                                 
